README.md

```diff
@@ -30,30 +30,44 @@ Spectator helps you get rid of all the boilerplate grunt work, leaving you with
 
 ## Table of Contents
 
+- [Features](#features)
```

Comment:
Why do we need Features and Table of Contents?

Reply:
hmm nice catch, I have no idea why this changed. I will remove it

Reply:
ahh it was my previewer trying to be "helpful"